doc/040_backup.rst

```diff
@@ -276,36 +276,53 @@ suffix the size value with one of ``k``/``K`` for kilobytes, ``m``/``M`` for meg
 Including Files
 ***************
 
-By using the ``--files-from`` option you can read the files you want to back
-up from one or more folders. This is especially useful if a lot of files have
-to be backed up that are not in the same folder or are maybe pre-filtered by
-other software.
+The options ``--files-from``, ``--files-from-verbatim`` and ``--files-from-raw``
+allow you to list the files to back up in one or more files. This is useful
+when a lot of files have to be backed up that are not in the same folder.
 
-For example maybe you want to backup files which have a name that matches a
-certain pattern:
+The argument passed to ``--files-from`` must be the name of a text file that
+contains one pattern per line. The file must be encoded as UTF-8, or UTF-16
+with a byte-order mark. Leading and trailing whitespace are removed from the
+patterns. Empty lines and lines starting with a ``#`` are considered comments.
+The patterns are expanded, when the file is read, by the Go function
+`filepath.Glob <https://golang.org/pkg/path/filepath/#Glob>`__.
+
+The option ``--files-from-verbatim`` has the same behavior as ``--files-from``,
+except that it contains literal filenames. It does not cause pattern expansion,
+comments are not allowed and whitespace is not trimmed off the filenames.
+Empty lines are still allowed, so that files can be grouped.
```

Comment:
Just slight adjustments to this paragraph:

~~~
The option ``--files-from-verbatim`` has the same behavior as ``--files-from``,
except that it contains literal filenames. It does not expand any patterns, filenames
are parsed verbatim. Lines starting with a ``#`` are not ignored, leading and trailing
whitespace is not trimmed. Empty lines are still allowed, so that files can be grouped.
~~~

Reply:
Ok.